shell.js

```diff
@@ -23,7 +23,7 @@ exports.pwd = common.wrap('pwd', _pwd);
 
 //@include ./src/ls
 var _ls = require('./src/ls');
-exports.ls = common.wrap('ls', _ls, {idx: 1});
+exports.ls = common.wrap('ls', _ls, {noGlob: true});
```

Comment:
I don't think `noGlob` is a real option (no `idx` value will have the same effect)

Also, would it be possible to modify `common.expand()` to take `all` as an argument? It might be cleaner to keep `glob` inside `common.js`